enterprise/server/tasksize/tasksize.go

```diff
@@ -322,7 +329,16 @@ func testSize(testSize string) (int64, int64) {
 // from the task such as test size and estimated compute units, but does not use
 // information about historical task executions.
 func Estimate(task *repb.ExecutionTask) *scpb.TaskSize {
-	props := platform.ParseProperties(task)
+	props, err := platform.ParseProperties(task)
+	if err != nil {
+		log.Warningf("Failed to parse task properties, using default estimation: %s", err)
```

Comment:
Let's make this a log.Info too, otherwise it'll affect app rollout if people are using invalid values

Reply:
fixed